vhost-device-scmi/src/main.rs

```diff
@@ -126,6 +126,11 @@ fn start_backend(config: VuScmiConfig) -> Result<()> {
         )
         .unwrap();
 
+        // Register devices such as "/dev/iio:deviceX" which can initiatively notify frontend to epoll handler.
+        // Then once there are data coming from these devices, it will occur event. (device_event=3)
```

Comment:
"Then once there is data coming from these devices, an event will be created. (device_event=3)"

Reply:
Well